### PR TITLE
1.5.0-beta02 Changelog

### PR DESCRIPTION
**Pull request by igordmn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,62 @@
...
+
+## Common
+
+### Features
+- [`androidx.compose.material.DropdownMenu` is available to use from common source set](https://github.com/JetBrains/compose-multiplatform-core/pull/738)
+- [`androidx.compose.material3.DropdownMenu` is available to use from common source set](https://github.com/JetBrains/compose-multiplatform-core/pull/737)
+- [`androidx.compose.material.AlertDialog` is available to use from common source set](https://github.com/JetBrains/compose-multiplatform-core/pull/708)
+- [`androidx.compose.material3.AlertDialog` is available to use from common source set](https://github.com/JetBrains/compose-multiplatform-core/pull/710)
+- [Add `PopupProperties.clippingEnabled` setting](https://github.com/JetBrains/compose-multiplatform-core/pull/740)
+- [Support `androidx.compose.material3.DatePicker`](https://github.com/JetBrains/compose-multiplatform-core/pull/717)
```
Comment (MatkovIvan) [inline CHANGELOG.md:11]: and TimePicker on iOS

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,62 @@
...
+- [Support `androidx.compose.material3.DatePickerDialog`](https://github.com/JetBrains/compose-multiplatform-core/pull/745)
+
+### Fixes
+- [Fix sending synthetic event on detach owner without recomposition](https://github.com/JetBrains/compose-multiplatform-core/pull/730)
```
Comment (MatkovIvan) [inline CHANGELOG.md:15]: It just allowed to remove workaround, no changes in public

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,62 @@
...
+
+### Features
+- [Support singleLine and `KeyboardAction`](https://github.com/JetBrains/compose-multiplatform-core/pull/699)
+
+### Fixes
+- [Fix memory leak in `ComposeUIViewController`](https://github.com/JetBrains/compose-multiplatform/issues/3201)
+- [Manage Kotlin native cache kind automatically based on Kotlin version](https://github.com/JetBrains/compose-multiplatform/pull/3477) (`kotlin.native.cacheKind=none` is no longer needed on Kotlin >= 1.9.0)
+- [Limit max `Dialog` and `Popup` size by safe area on iOS](https://github.com/JetBrains/compose-multiplatform-core/pull/732)
+- [`TextField`, Korean characters are not normally entered](https://github.com/JetBrains/compose-multiplatform/issues/3101)
+- [ColorMatrix value range for 5th column is incorrect on Skiko backed platforms](https://github.com/JetBrains/compose-multiplatform/issues/3461)
```
Comment (MatkovIvan) [inline CHANGELOG.md:30]:
```suggestion
- [`ColorMatrix` value range for 5th column was incorrect on Skiko backed platforms](https://github.com/JetBrains/compose-multiplatform/issues/3461)
```

File: CHANGELOG.md (hunk 1 above)
Comment (igordmn) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (igordmn) [reply CHANGELOG.md]: Fixed

**Review by pjBooms — APPROVED** [review]

**Pull request merged**